prov/efa/src/efa_user_info.c

```diff
@@ -354,6 +355,31 @@ int efa_user_info_alter_rxr(struct fi_info *info, const struct fi_info *hints)
 
 			info->domain_attr->mr_mode |= FI_MR_HMEM;
 
+			/* TODO Loop over every HMEM interface and determine if we can support atomics */
+			support_atomic = true;
```

Comment:
wrap the whole section under `if (info->caps & FI_ATOMIC)`

Reply:
I don't think we want to do this b/c then we would only be checking atomic support when the user requests atomics in their hint structure, and blindly supporting it in all other cases.

Reply:
discussed offline, no action required.